guides/user-manual/api_auth.rst

```diff
@@ -58,25 +78,23 @@ where the last part is ``<user name>:<password>``, base64 encoded
 
 For more information on Basic Authorization headers on the client side, see for example here: https://en.wikipedia.org/wiki/Basic_access_authentication
 
-The authorization header authentication method is only available for JSON-RPC and REST APIs, and only if using M&M Web Services as an endpoint (i.e. the URL is <server>/mmws/api or <server>/mmws/json)
-
 .. note::
   Remember to configure HTTPS for the mmws site for this authentication method since the username and password can easily be extracted for anyone listening in on the communication.
 
```

Comment:
Maybe we want to add a note here that Micetro can be configured to disable Basic Authorization header by adding `<AuthorizationMethods value="" />` into the preference file for mmWS?